app/services/gpo_confirmation_uploader.rb

```diff
@@ -33,19 +33,15 @@ def clear_confirmations(confirmations)
 
   def remote_path
     timestamp = @now.strftime('%Y%m%d')
-    File.join(env.usps_upload_sftp_directory, "batch#{timestamp}.psv")
+    File.join(IdentityConfig.store.usps_upload_sftp_directory, "batch#{timestamp}.psv")
   end
 
   def sftp_config
     [
-      env.usps_upload_sftp_host,
-      env.usps_upload_sftp_username,
-      password: env.usps_upload_sftp_password,
-      timeout: env.usps_upload_sftp_timeout.to_i,
+      IdentityConfig.store.usps_upload_sftp_host,
+      IdentityConfig.store.usps_upload_sftp_username,
+      password: IdentityConfig.store.usps_upload_sftp_password,
+      timeout: IdentityConfig.store.usps_upload_sftp_timeout,
     ]
   end
-
-  def env
-    AppConfig.env
-  end
```

Comment:
this is totally an artifact from Reek lol

Reply:
"How _dare_ you invoke this method twice 😠 "